.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: Bloop on Windows
+on:
+  push:
+jobs:
+  unix:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: olafurpg/setup-scala@v4
+        with:
+          java-version: graalvm@
+      - run: git submodule update --init --recursive
```

Comment:
`actions/checkout` has a `submodules` parameter that recursively clones also submodules, could that help?

Reply:
or run the step separately

```yml
      - run: git submodule update --init --recursive
```

https://github.com/scalacenter/bloop/blob/0536c40217da480c4ae39f3b1cf47be77f144f14/.github/workflows/bloopgun.yml#L16